pom.xml

```diff
@@ -91,6 +91,8 @@
                 <configuration>
                     <excludes>
                         <exclude>**/Main*</exclude>
+                        <exclude>**/model/Fruit*</exclude>
+                        <exclude>**/model/Transaction*</exclude>
```

Comment:
Ksenia today said we shouldn't write path to our classes, but if it works in your case , its ok)